include/mbedtls/ssl.h

```diff
@@ -2463,15 +2463,15 @@ int mbedtls_ssl_read( mbedtls_ssl_context *ssl, unsigned char *buf, size_t len )
  *                 or MBEDTLS_ERR_SSL_WANT_WRITE or MBEDTLS_ERR_SSL_WANT_READ,
  *                 or another negative error code.
  *
- * \note           If this function returns something other than a positive
+ * \note           If this function returns something other than 0, a positive
  *                 value or MBEDTLS_ERR_SSL_WANT_READ/WRITE, the ssl context
  *                 becomes unusable, and you should either free it or call
  *                 \c mbedtls_ssl_session_reset() on it before re-using it for
  *                 a new connection; the current connection must be closed.
  *
  * \note           When this function returns MBEDTLS_ERR_SSL_WANT_WRITE/READ,
  *                 it must be called later with the *same* arguments,
- *                 until it returns a positive value.
+ *                 until it returns a value greater that or equal to 0.
```

Comment:
It should be `greater than or` here.

Reply:
You mean that the sentence should be `... until it returns a value greater then or 0`?

Reply:
`until it returns a value greater than or equal to 0`